### PR TITLE
markezine.jp

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/112415

<details><summary>Screenshot:</summary>

Снимок экрана 2022-03-15 в 17 17 55

Снимок экрана 2022-03-15 в 17 18 55

Снимок экрана 2022-03-15 в 17 18 06

</details><br/>

### CODE REVIEW

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -476,6 +476,7 @@ appllio.com#?#.sidebar > .region-inner > .block.odd:has(div[id^="div-gpt-ad"])
 !
 pokemon-matome.net#?#.blogparts_freeArea:has(a[href^="https://www.amazon.co.jp/exec/"])
 753ya.net#?#.grid_post-box ins.adsbygoogle:upward(.grid_post-box)
+markezine.jp##.l-sidebar_inner > .c-secondarysection:has(> .c-secondarysection_header > p.c-secondarysection_heading:contains(スポンサーサイト))
```
Comment (Alex-302) [inline JapaneseFilter/sections/css_extended.txt:479]: `#?#`

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -60,6 +60,7 @@ rebaltica.lv,pokemongo-master.com,cricbuzz.com,bizmates.jp,stardoll.com,yakugaku
 !
 nepisirsem.com###topshare
 conan-mobile.jp###snsbar
+markezine.jp##li[class^="c-article"] > a[href^="https://note.com/"]
```
[the remark this answers is not in the document]
Comment (Alex-302) [reply SocialFilter/sections/specific.txt:63]: Agree.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/tweaks.txt [hunk 3]
```diff
@@ -9,6 +9,8 @@
 !######### CSS tweaks #########
 !##############################
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/112415
+markezine.jp#$#.l-sidebar_inner > .ad-job-board-wrap + .c-secondarysection  { margin-top: 0px !important; }
```
[the remark this answers is not in the document]
Comment (zloyden) [reply AnnoyancesFilter/sections/tweaks.txt:13]: As I can see, there are typo and missed tags in the rule.
`markezine.jp#$#.l-sidebar_inner > div.ad-job-board-wrap + div.c-secondarysection { margin-top: 0px !important; }`

File: JapaneseFilter/sections/css_extended.txt (hunk 1 above)
Comment (zloyden) [reply JapaneseFilter/sections/css_extended.txt:479]: Missed tags after `>`

File: SocialFilter/sections/specific.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (zloyden) [reply SocialFilter/sections/specific.txt:63]: @Yuki2718 I would say that your rule is ok here

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**